database/localDB/main/CreateTable.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import "github.com/bohdanlisovskyi/Golang-252-Telegram-Bot-Game/core/loger"
+
+type Planet struct {
```

Comment:
What the different between postgres.Planet and this one

Reply:
Structures are the same. Package main consist code for creating database on your local machine (in case any one need it) for testing.